.github/workflows/build-plugin-archive.yml

```
@@ -202,16 +215,27 @@ jobs:
           composer global require inpsyde/wp-translation-downloader
           composer wp-translation-downloader:download
 
+      - name: Run PHP Scoper
+        if: steps.composer-tools.outputs.php-scoper == '0'
+        run: |
+          php-scoper add-prefix --force --output-dir build
+          composer --working-dir=build dump-autoload -o
+          sed -i "s/'__composer_autoload_files'/\'__composer_autoload_files_${{ github.sha }}'/g" "build/vendor/composer/autoload_real.php"
+      - name: Move unchanged code to build directory
```

On the first read, the "unchanged" part of the step name is confusing, IMO. Would it be possible to run the previous step in the current directory instead of in `build/`? This would make this step superfluous. If not, let's try to improve the name.
```suggestion
      - name: Move code to the build directory

```
New line also added, again.

---

What's happening here:

 - If php-scoper is in use, it has processed the code from `cwd` into `build/`
 - If php-scoper it **NOT** in use, then the code is _unchanged_ and `build/` is empty
 
 Therefore, we need this step to make sure that in all scenarios, we can continue with the workflow from `build/`

---

Yes, yes, I understand the code. My question was if PHP-Scoper could work in the current directory, making this step superfluous for products not using the tool.

---

To my knowledge, php-scoper cannot work in-place, so the output dir must differ. An alternative could be to let scoper write to a temp directory and then replace the current dir's contents.
This might make it a bit clearer to chain additional tools since we do not switch directories at some point. But on a technical level, it's not much of a difference. What do you prefer?

---

>What do you prefer?

Let's keep things as-is, but maybe change the step name as suggested.